stories/Common/ValidationInput.stories.tsx

```
@@ -0,0 +1,74 @@
+
+export const Success: Story = {
+  args: {
+    messageType: "success",
+    message: "사용 가능한 아이디입니다.",
+  },
+};
+
+export const Error: Story = {
+  args: {
+    messageType: "error",
+    message: "이미 사용 중인 아이디입니다.",
```

Biome (2.1.2)

[error] Do not shadow the global "Error" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)